disko.py

```diff
@@ -1,4 +1,14 @@
-from src.disko.image_collector import ImageCollector
+from src.disko.image_management.image_view import ImageRegistryManager
 
-collector = ImageCollector()
-collector.collect_images()
+# main function
+def main():
+    # Database file path
+    db_file = 'image_data.db'
+
+    # Run GUI
+    gui = ImageRegistryManager(db_file)
+    gui.run()
```

Comment:
There's a different approach we would like to achieve.
The way it is built in your commit now, is that the controlling of what happens is I the view. It should be in the controller.
The controller should gather all the information and send it to the view.
The way you have structured it here is that the view is in charge. We should change it.
The idea is that the view should be replaceable tomorrow. So no logic should happen there. All should happen in the controller.